velox/dwio/dwrf/test/ColumnWriterTest.cpp

```diff
@@ -1267,12 +1267,10 @@ TEST_F(ColumnWriterTest, TestMapWriterDuplicatedInt64Key) {
   auto batch = b::create(
       *pool, {typename b::row{typename b::pair{5, 3}, typename b::pair{5, 4}}});
 
-  EXPECT_THAT(
-      ([&]() { testMapWriter<T, T>(*pool, batch, /* useFlatMap */ true); }),
-      Throws<
-          facebook::velox::dwio::common::exception::LoggedException>(Property(
-          &facebook::velox::dwio::common::exception::LoggedException::message,
-          HasSubstr("Duplicated key in map: 5"))));
+  VELOX_ASSERT_THROW_IMPL(
```

Comment:
VELOX_ASSERT_THROW is not sufficient here? I'd think we don't want to use VELOX_ASSERT_THROW_IMPL.
```
VELOX_ASSERT_THROW((testMapWriter<T, T>(*pool, batch, /* useFlatMap */ true)), "Duplicated key in map: 5");
```
The type of exception is probably not that important?

Reply:
You are right. `class LoggedException : public velox::VeloxException`. `VELOX_ASSERT_THROW` is sufficient here.